i18n/management/commands/i18n_message_file_sync.py

```diff
@@ -0,0 +1,86 @@
+# pylint: disable=missing-docstring
+import glob
+import os
+import json
+
+from django.core import management
+from django.core.management.base import BaseCommand
+from django.utils.translation import to_locale
+from django.core.files.storage import FileSystemStorage
+
+from i18n.management.crowdin import Crowdin
+from i18n.management.utils import log, get_non_english_language_codes
+from i18n.utils import I18nFileWrapper
+
+
+class Command(BaseCommand):
+
+    # Retrieves all django.po files from Crowdin and
+    # saves them locally
+    def download_message_file(self):
+        # Download changes!
```

Comment:
A lot of this code is duplicated from download_translations. Is there a way to either parameterize that function or extract some of the duplicate logic?

Reply:
For sure, I'll update the function with an optional argument